docs/presentation/index.html

```diff
@@ -113,6 +119,101 @@
+                    &lt;/water&gt;
+                  &lt;/default&gt;
+                &lt;/recipe&gt;
+              </code>
+            </pre>
+
+            <aside class="notes">The recipe begins by defining the Default Stage. This will encapsulate the baseline settings for each of our nutrients and metrics. When we give a metric, we can define a tolerable range, for what's acceptable for one of our nutrients, or we can specify a precise value.</aside>
+          
+          </section>
+
+          <section>
+            <h2>Adding Additional Stages</h2>
+            <pre>
+              <code class="xml no-max-height" data-trim>
+                &lt;recipe&gt;
+                  &lt;default&gt;
+                    &lt;air&gt;
+                      &lt;temperature min=&quot;18&quot; max=&quot;24&quot; /&gt;
+                    &lt;/air&gt;
+                    &lt;water&gt;
+                      &lt;ph min=&quot;6.8&quot; max=&quot;6.8&quot; /&gt;
+                    &lt;/water&gt;
+                  &lt;/default&gt;
+                  &lt;vegetative&gt;
+                    &lt;air&gt;
+                      &lt;temperature min=&quot;12&quot; max=&quot;15&quot; /&gt;
+                    &lt;/air&gt;
+                  &lt;/vegetative&gt;
+                &lt;/recipe&gt;
+              </code>
+            </pre>
+
+            <aside class="notes">Each additional stage we add builds upon the default stage. We only include the changes we want for the new stage.</aside>
+          </section>
+
+          <section>
+            <h2>Timing and Duration</h2>
+            <pre>
+              <code class="xml no-max-height" data-trim>
+                &lt;recipe&gt;
+                  &lt;default&gt;
+                    &lt;air&gt;
+                      &lt;temperature min=&quot;18&quot; max=&quot;24&quot; /&gt;
+                    &lt;/air&gt;
+                    &lt;water&gt;
+                      &lt;ph min=&quot;6.8&quot; max=&quot;6.8&quot; /&gt;
+                    &lt;/water&gt;
+                  &lt;/default&gt;
+                  &lt;vegetative&gt;
+                    &lt;air&gt;
+                      &lt;temperature min=&quot;12&quot; max=&quot;15&quot; /&gt;
+                    &lt;/air&gt;
+                  &lt;/vegetative&gt;
+                &lt;/recipe&gt;
+              </code>
+            </pre>
+
+            <aside class="notes">
+              For each of the non-default stages we can include Start and End times, based on a 24 hour cycle.<br>
+              Or.. even add day duration for how long the stage should be defined.<br>
+              This needs to be discussed during the call on wednesday, or earlier.
+            </aside>
+          </section>
+
+          <section>
+            <h2>Recipe Conversions!</h2>
+            <p class="fragment">Include one recipe for tomatofest, to show ease of use</p>
+            <p class="fragment">Include a second recipe to show how its depth and scalability</p>
```

Comment:
Might be able to do the demo within the presentation using this https://github.com/iamvdo/Reveal-liveCoding

Reply:
Awesome! it'd be good to try it out, but if it works, I'd say lets do it.